index.html

```diff
@@ -5,157 +5,190 @@
-                </div>
-                <div class="container4">
-                    <button class="button1">See Projects</button>
-                </div>
-                <div class="container2">
-                    <div class="Data"><h1>Profesional Art Printing Data</h1></div>
-                    <div class="para1"><p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard.</p></div>
-                    <ul class="container3">
-                        <li class="item4">css</li>
-                        <li class="item5">bootstrap</li>
-                        <li class="item6">Ruby</li>
-                    </ul>
+                    <div class="listo">
+                        <ul class="a">
+                            <li class="item1">css</li>
+                            <li class="item2">html</li>
+                            <li class="item3">bootstrap</li>  
+                            <li class="item2">Ruby</li>  
+                        </ul>
+                        <button class="button">See Projects</button>
+                    </div>
                 </div>
-                <div class="container4">
-                    <button class="button1">See Projects</button>
+                <div class="containo">
+                        <div class="container2">
+                            <div class="hovee">
+                                <div class="Data"><h1>Profesional Art Printing Data</h1></div>
+                                <div class="para1"><p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard.</p></div>
+                                <ul class="container3">
+                                    <li class="item4">css</li>
+                                    <li class="item5">bootstrap</li>
+                                    <li class="item6">Ruby</li>
+                                </ul>
+                            </div>
+                                <div class="container4">
+                                    <button class="button1">See Projects</button>
+                                </div>
+                        </div>
+                        <div class="container2">
+                            <div class="hovee">
+                                <div class="Data"><h1>Profesional Art Printing Data</h1></div>
+                                <div class="para1"><p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard.</p></div>
+                                <ul class="container3">
+                                    <li class="item4">css</li>
+                                    <li class="item5">bootstrap</li>
+                                    <li class="item6">Ruby</li>
+                                </ul>
+                            </div>
+                            <div class="container4">
```

Comment:
[OPTIONAL] ⭐ This is an optional change but recommended. I strongly suggest that you keep your code as simplest as possible. Wrapping other elements like `h1` or `p` tags just for styling into a `div` is not recommended. This makes your code less readable, not very semantic and it is not screen-reader friendly. Remember that you always can style those tags directly.

Please also remember that thee `h`1` tag is the most important title on your page. You should avoid using several of these.